custom_components/tahoma/cover.py

```diff
@@ -252,15 +252,21 @@ def icon(self):
 
     async def async_open_cover(self, **_):
         """Open the cover."""
-        await self.async_execute_command(self.select_command(*COMMANDS_OPEN))
+        if self._is_horizontal():
+            await self.async_execute_command(COMMAND_DEPLOY)
+        else:
+            await self.async_execute_command(self.select_command(*COMMANDS_OPEN))
 
     async def async_open_cover_tilt(self, **_):
         """Open the cover tilt."""
         await self.async_execute_command(self.select_command(*COMMANDS_OPEN_TILT))
 
     async def async_close_cover(self, **_):
         """Close the cover."""
-        await self.async_execute_command(self.select_command(*COMMANDS_CLOSE))
+        if self._is_horizontal():
+            await self.async_execute_command(COMMAND_UNDEPLOY)
```

Comment:
Don't we need to add these commands to `supported_features`

Reply:
To be clean indeed. I will add them.
But as they are always present with `closure*`commands is already working.

Reply:
Fixed.